linters/tslint.json

```diff
@@ -0,0 +1,154 @@
+{
+  "extends": ["tslint-eslint-rules"],
+  "rules": {
+
+    // RECOMMENDATIONS
+
+    "interface-name": [true, "never-prefix"],     // Require interface names to not have an “I” prefix
+    "deprecation": true,                          // Warn when deprecated APIs are used
+
+    // MODULES <https://github.com/thenerdery/javascript-standards#modules>
+
+    "no-duplicate-imports": true,                 // Disallow duplicate imports
+
+    // ARROW FUNCTIONS <https://github.com/thenerdery/javascript-standards#arrow-functions>
+
+    "prefer-arrow-callback": 2,                   // Require arrow functions as callbacks
+    "arrow-spacing": [2, {                        // Require space before/after arrows
+      "before": true,
+      "after": true
+    }],
+
+    // VARIABLES <https://github.com/thenerdery/javascript-standards#variables>
+
+    "variable-name": [                            // Check variable names for various errors
+      true,
+      "ban-keywords",
+      "check-format",
+      "allow-leading-underscore",
+      "allow-pascal-case"
+    ],
+    "one-variable-per-declaration": true,         // Disallow multiple variable definitions in the same declaration statement
+    "no-duplicate-variable": [                    // Disallow duplicate variable declarations in the same block scope, including parameters
+      true,
+      "check-parameters"
+    ],
+    "no-unused-expression": true,                 // Disallow unused expression statements
+    "no-shadowed-variable": true,                 // Disallow shadowing variable declarations
+    "no-unused-variable": true,                   // Disallow unused imports, variables, functions and private class members
+    "no-use-before-declare": true,                // Disallow usage of variables before their declaration
+    "no-var-keyword": true,                       // Disallow usage of the var keyword
+    "prefer-const": true,                         // Require const when not reassigned
+
+    // OBJECTS <https://github.com/thenerdery/javascript-standards#objects>
+
+    "object-literal-shorthand": true,             // Enforce use of ES6 object literal shorthand
+
+    // TYPES <https://github.com/thenerdery/javascript-standards#types>
+
+    "typedef": [                                  // Require type definitions to exist
+      true,
+      "parameter",
+      "property-declaration"
+    ],
+    "typedef-whitespace": [                       // Enforce one space after type information
+      true,
+      {
+          "call-signature": "nospace",
+          "index-signature": "nospace",
+          "parameter": "nospace",
+          "property-declaration": "nospace",
+          "variable-declaration": "nospace"
+      }
+    ],
+    "no-any": true,                               // Disallow usage of any as a type declaration
```

Comment:
If the developer didn't know the type for a external library object. We would tell them to use the `unknown` type?

Reply:
I think `unknown` is only a TypeScript 3 thing. I noticed on our current project a library has types but there not complete and some are wrong so we have been adding `any` for those because we are not on TypeScript 3.